src/gui/MainWindow.cpp

```diff
@@ -971,7 +971,10 @@ void MainWindow::trayIconTriggered(QSystemTrayIcon::ActivationReason reason)
 void MainWindow::hideWindow()
 {
     saveWindowInformation();
-#ifndef Q_OS_MAC
```

Comment:
Why remove this guard for MacOS?

Reply:
Good catch... @phoerious said in issue #1595 that isn't probably an ancient code from the time Mac didn't have a tray icon, and I remember doing a git blame, but it looks like I did it wrong, because it's rather the opposite... it was introduced in 8d6db27b3430543e6f262a566be0cbd0331a404e when the tray icon on Mac was **introduced**... I will add it again, along with a TODO asking for further comments.